docs/source/dataframe-create.rst

```diff
@@ -58,22 +46,43 @@ In particular column access on a dask.dataframe backed by a ``bcolz.ctable``
 will only read the necessary columns from disk.  This can provide dramatic
 performance improvements.
 
+.. _BColz: http://bcolz.blosc.org/
+
 From Castra
-~~~~~~~~~~~
+-----------
 
 Castra_ is a tiny, experimental partitioned on-disk data structure designed to
 fit the ``dask.dataframe`` model.  It provides columnstore access and range
-queries along the index.  It is also a very small project (roughly 400 lines).
+queries along the index.  It is also a very small project (roughly 400 lines)
+and not actively maintained; use at your own risk.
 
 .. code-block:: Python
 
    >>> from castra import Castra
    >>> c = Castra(path='/my/castra/file')
    >>> df = c.to_dask()
 
+.. _Castra: http://github.com/Blosc/castra
+
+
+From Bags
+---------
+
+You can create a dask.dataframe from a dask bag.
+
+.. autosummary:: dask.bag.core.Bag.to_dataframe
+
+Using ``dask.delayed``
+----------------------
+
+You can create a plan to arrange many Pandas frames into a sequence with normal
+for loops using :doc:`dask.delayed<imperative-overview>` and then convert these
+into a dask dataframe later.  See :doc:`documentation on using dask.delayed with
+collections<imperative-collections>`.
```

Comment:
What's the use case of using `dask.delayed` instead of `Bag.to_dataframe` here? If you're already using `dask.delayed` elsewhere? It's not immediately clear to me.